sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -39,6 +39,9 @@
 from .test_perf_utils import calc_compilation, get_times, perf_data_gen_fixed_len
 
 
+global_vars = {'time': time, 'np': np}
```

Comment:
It is not necessary to make this variable global. I meant different thing: passing to exec() globals from the current module - consider Python function globals().